src/library_html5_webgl.js

```diff
@@ -323,6 +323,7 @@ var LibraryHtml5WebGL = {
 #if MAX_WEBGL_VERSION >= 2
     '_webgl_enable_WEBGL_draw_instanced_base_vertex_base_instance',
 #endif
+    '_webgl_enable_WEBGL_multi_draw',
```

Comment:
Can you gate all of the five above deps to be present only `#if GL_SUPPORT_SIMPLE_ENABLE_EXTENSIONS` - looks like that was missing from before. Not sure, but that might help with the minimal runtime code size test.